app/api/helpers/payment.py

```diff
@@ -117,119 +114,56 @@ def capture_payment(order_invoice, currency=None, credentials=None):
 
 
 class PayPalPaymentsManager(object):
-    api_version = 93
+    """
+    Class to manage payment through Paypal Braintree.
+    """
 
     @staticmethod
-    def get_credentials(event=None, override_mode=False, is_testing=False):
+    def get_credentials():
         """
-        Get Paypal credentials from settings module.
-        :param event: Associated event.
-        :param override_mode:
-        :param is_testing:
-        :return:
+        Initialize and return the Braintree payment gateway.
+        :return: Braintree gateway instance.
         """
-        if event and represents_int(event):
-            event = safe_query(db, Event, 'id', event, 'event_id')
+        # Use Sandbox by default.
         settings = get_settings()
-        if not override_mode:
-            if settings['paypal_mode'] and settings['paypal_mode'] != "":
-                if settings['paypal_mode'] == 'live':
-                    is_testing = False
-                else:
-                    is_testing = True
-            else:
-                return None
+        paypal_braintree_access_token = settings['paypal_braintree_sandbox_access_token']
 
-        if is_testing:
-            credentials = {
-                'USER': settings['paypal_sandbox_username'],
-                'PWD': settings['paypal_sandbox_password'],
-                'SIGNATURE': settings['paypal_sandbox_signature'],
-                'SERVER': 'https://api-3t.sandbox.paypal.com/nvp',
-                'CHECKOUT_URL': 'https://www.sandbox.paypal.com/cgi-bin/webscr',
-                'EMAIL': '' if not event or not event.paypal_email or event.paypal_email == "" else event.paypal_email
-            }
-        else:
-            credentials = {
-                'USER': settings['paypal_live_username'],
-                'PWD': settings['paypal_live_password'],
-                'SIGNATURE': settings['paypal_live_signature'],
-                'SERVER': 'https://api-3t.paypal.com/nvp',
-                'CHECKOUT_URL': 'https://www.paypal.com/cgi-bin/webscr',
-                'EMAIL': '' if not event or not event.paypal_email or event.paypal_email == "" else event.paypal_email
-            }
-        if credentials['USER'] and credentials['PWD'] and credentials['SIGNATURE'] and credentials['USER'] != "" and \
-                credentials['PWD'] != "" and credentials['SIGNATURE'] != "":
-            return credentials
-        else:
-            return None
+        # Switch to production if environment is production.
+        if settings['app_environment'] == Environment.PRODUCTION:
```

Comment:
Why do we need this?

Reply:
Because for production we have to use different access token. For development instance we will use the sandbox access token which is specially for testing. In the actual production software we are going to use the real access code.